example_agents/acme_r2d2/components.yaml

```diff
@@ -7,15 +7,15 @@ specs:
         type: Instance
         argument_set:
             type: ArgumentSet
-            kwargs:
-                environment: spec:environment
-                policy: spec:policy
-                dataset: spec:dataset
-                trainer: spec:trainer
-                AcmeRun: spec:AcmeRun
-                network: spec:network
+            args:
+                - spec:environment
```

Comment:
Yep. The ArgumentSet provides both a way to get its args/kwargs without hydrating, and also a way to get the recursively hydrated args/kwargs. The Instance Component is responsible for using the latter functionality when it needs it instead of the former.